src/python/pants/option/global_options.py

```diff
@@ -3,6 +3,9 @@
 
 from __future__ import annotations
 
+import enum
```

Comment:
there is already an import on line 13.. maybe update that import ?
<img width="689" alt="Screen Shot 2021-01-28 at 11 07 58 PM" src="https://user-images.githubusercontent.com/1268088/106243133-aa5f0800-61bd-11eb-8b94-a86960443971.png">

Reply:
Yeah but I didn't want to introduce `@unique` into the global namespace without the `enum` prefix. I could then "fix" subclasses of `Enum` to be `enum.Enum`, but that felt clunky.